hs_geographic_feature_resource/templates/pages/geographicfeatureresource.html

```diff
@@ -0,0 +1,235 @@
+{#                </a></td>#}
+{#            {% endif %}#}
+{#        {% endfor %}#}
+
+
+        {# Add ncdump section #}
+{#        {% if cm.files.all %}#}
+            {# tab body for ncdump #}
+{#            <div id="ncdumpText">#}
+{##}
+{#                {% for f in cm.files.all %}#}
+{#                    {% if f.resource_file.name|slice:"-3:" == 'txt'%}#}
+{#                        <div id="showNcdump">#}
+{#                            <h3 style="margin-top:40px"> NetCDF Header Info</h3>#}
+{#                            <p>{{ f.resource_file.name|slice:"33:" }}</p>#}
+{#                            {% if f.resource_file.size <= 5000000 %}#}
+{#                                <textarea readonly rows="15" style="margin-bottom:20px;border: 1px solid #ddd;width:100%">{{ f.resource_file.read}}</textarea>#}
+{#                            {% else %}#}
+{#                                <textarea readonly rows="5" style="margin-bottom:20px;border: 1px solid #ddd;width:100%">The size of the netCDF header information text file is too large for loading on the page.</textarea>#}
+{#                        {% endif %}#}
+{#                        </div>#}
+{#                    {% else %}#}
+{#                        {% if page.perms.change %}#}
+{#                            <div id="createNcdump" style="margin-bottom: 25px">#}
+{#                                <h3 style="margin-top:40px"> NetCDF Header Info</h3>#}
+{#                                <p>Click the button to create a new header info .txt file for the resource.</p>#}
+{#                                <a id="create-ncdump-btn" type="button" class="btn btn-success" href="{% url "nc_tools:create_ncdump" cm.short_id %}">#}
+{#                                    <span class="button-label"> Create header info file</span>#}
+{#                                </a>#}
+{#                            </div>#}
+{#                        {% endif %}#}
+{#                    {% endif %}#}
+{#                {% endfor %}#}
+{##}
+{#            </div>#}
+{#        {% endif %}#}
+
+
+    {% endblock %} {# extended block ends here #}
+
+
+{%  endblock %}
+
+
+{% block extra_js %}
+
+{{ block.super }}
+
+{# put resource specific js below #}
+<script type="text/javascript">
+    // original coverage meta check
+    function check_ori_meta_status(){
+        var form_action = $("form[id='id-originalcoverage']").attr('action');
+        if (form_action.indexOf('update-metadata') > -1) {
+            var new_action = form_action.replace('update-metadata', 'delete-metadata');
+            var modal_body_text = '<strong>Are you sure you want to delete this metadata element?</strong>'
+            $("#delete-original-coverage-element-dialog").find(".modal-body").html(modal_body_text);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").attr('href',new_action);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").show()
```

Comment:
these lines can be simplified into a single jQuery selector like `$("#delete-original-coverage-element-dialog .modal-footer a")`